.github/workflows/scripts/build_and_inspect_inputs.sh

```diff
@@ -0,0 +1,80 @@
+"""
+Functionality for generating and inspecting CSV-Ws for the input files.
+"""
+
+echo "::set-output name=has_outputs::false"
+
+echo "Input argument: $1"
+IFS=', ' read -r -a added_modified_files <<< "$1"
+echo "added_modified_files: ${added_modified_files[@]}"
+
+for file in "${added_modified_files[@]}"; do
+  echo "${file}"
+  file_path="${file%.*}"
+  file_name="${file_path##*/}"
+  file_extension="${file##*.}"
+  out_path="out/${file_path}/"
+  
+  # Detects the top folder from the file path. E.g. out/ is the top folder when the path is out/sub-folder/my-data.csv
+  top_folder=$(echo "$file_path" | cut -d "/" -f1)
+
+  echo "file_path: ${file_path}"
```

Comment:
Maybe not the most important issue right now, but is there a better way to do logging with github actions? I'd ideally like it if we could log this at `debug` level so it wouldn't show up in most instances.